### PR TITLE
Experiment/space

### PR DESCRIPTION
**Pull request by topological-ragnar**
spaaaaaace

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
 var environment = require('./environment')
+var apod = require('nasa-apod')
+var $ = require('jquery')
+
+var randomDate = function randomDate(start, end) {
+    return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+}
```
Comment (data-doge) [inline js/index.js]: feel like this either belongs in `./js/services/random-date.js` or we can use this [already built module](https://www.npmjs.com/package/random-date)

File: js/index.js [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+var date = randomDate(new Date(2012, 0, 1), new Date())
+
+apod(date).then(function(data) {
+    $('#body').css("background-image","url("+data.hdurl+")")
+    $('#body').css("background-position","center")
+    $('#body').css("background-size","cover")
+    $('#body').css("background-repeat","no-repeat")
+    console.log(data.url)
+})
+
```
Comment (data-doge) [inline js/index.js]: feel like this should be a submodule inside of `./js/hud`

File: js/index.js [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+var $ = require('jquery')
+
+var randomDate = function randomDate(start, end) {
+    return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+}
+
+var date = randomDate(new Date(2012, 0, 1), new Date())
+
+apod(date).then(function(data) {
+    $('#body').css("background-image","url("+data.hdurl+")")
```
Comment (data-doge) [inline js/index.js]: there'll only be on `<body>` in `index.html` so we can omit the ID, and do something like

``` javascript
var $body = $('body')
$body.css('background-image', 'url(' + data.hdurl + ')'
...
...
```

File: index.html [hunk 4]
```diff
@@ -8,7 +8,7 @@
     <link href="./css/normalize.css" rel="stylesheet" />
     <link href="./css/index.css" rel="stylesheet" />
   </head>
-  <body>
+  <body id='body'>
```
Comment (data-doge) [inline index.html]: see comments in `js/index.js`, can remove `id` from `<body>`

File: js/index.js [hunk 5]
```diff
@@ -1,4 +1,21 @@
...
+    $('#body').css("background-position","center")
+    $('#body').css("background-size","cover")
+    $('#body').css("background-repeat","no-repeat")
+    console.log(data.url)
```
Comment (data-doge) [inline js/index.js]: remove log?

File: js/index.js (hunk 1 above)
Comment (topological-ragnar) [inline js/index.js]: let's use the module

File: js/index.js (hunk 2 above)
Comment (topological-ragnar) [inline js/index.js]: ya aight

File: js/index.js (hunk 3 above)
Comment (topological-ragnar) [inline js/index.js]: nice

File: js/index.js (hunk 5 above)
Comment (topological-ragnar) [inline js/index.js]: image

File: js/environment/index.js [hunk 6]
```diff
@@ -54,8 +52,7 @@ module.exports = {
   move: function(dt) {
```
Comment (data-doge) [inline js/environment/index.js]: wondering if a more descriptive name for this method might be `updateParticlePositions` or something of that sort

Comment (topological-ragnar) [inline js/environment/index.js]: like it

File: js/hud/index.js [hunk 7]
```diff
@@ -7,6 +10,14 @@ var hud = {
...
     this.buttons.init(env)
     this.sketchpad.init()
+    var date = randomDate(new Date(2012, 0, 1),new Date())
+    console.log(date)
+    apod(date).then(function(data) {
+        $('body').css("background-image","url("+data.hdurl+")")
+        $('body').css("background-position","center")
+        $('body').css("background-size","cover")
+        $('body').css("background-repeat","no-repeat")
+    })
```
Comment (data-doge) [inline js/hud/index.js]: feels like this should be in a method called `setBackgroundImage` and then called in `init`?

**Pull request merged**